Gemfile

```diff
@@ -31,6 +31,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'whenever', require: false
 gem 'xmlenc', '~> 0.6.4'
 gem 'activerecord-session_store', '1.0.0.pre'
+gem 'rqrcode'
```

Comment:
I think we've been trying to keep gems in alphabetical order to find them more easily in the list. That `activerecord-session_store` gem placement is an anomaly and should be fixed in a separate PR.